strict_test.go

```diff
@@ -21,7 +21,7 @@ var _ = Describe("Negotiator", func() {
 		var err error
 		r, err = http.NewRequest("POST", "http://example.com/", nil)
 		Expect(err).
-			NotTo(HaveOccured())
+			NotTo(HaveOccurred())
```

Comment:
Aah, this is so annoying. If I had the time, I'd eradicate ginkgo/gomega from the tests altogether and use plain old `testing`.

Thanks for fixing this.